spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
@dkuebric 

> If I understand your last proposal right, it fails to allow a case where traces are started at "edge" create_span calls, but not started at "interior" calls, because it is a global property of the tracer and the two categories are not distinguished. 

No.

> So it still requires instrumentors to do if-guards in all their subsequent instrumentation logic.

No.

Given

def process_initialize
    sampling = Sampling.NO_NEW_TRACES
    tracer = Tracer.create(sampling)

it means that anywhere (interior or edge) in that process where `tracer.create_span(null)` is called a new trace is not created. Because in lei of an existing trace being present the tracer for this process is setup with a sampling decision of always false.

This avoids the need for if-guards through the code.